docs/changelog.rst

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 **Bug fixes**
 
 - Fix `loadinfrastructure` condition when adding to infra without `eid-field` option (#4328)
+- Fix error 500 at intervention creation when target is not a report (suricate workflow)
```

Comment:
```suggestion
- Fix intervention creation when target is not a report (suricate workflow)
```
No need to precise that exception raise 500 because it's always the case